dist/stat_data.py

```diff
@@ -526,6 +526,9 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_prepare_active', 'prepared transactions currently active'),
     TxnStat('txn_prepare_commit', 'prepared transactions committed'),
     TxnStat('txn_prepare_rollback', 'prepared transactions rolled back'),
+    TxnStat('txn_prepared_updates_active', 'prepared active updates'),
+    TxnStat('txn_prepared_updates_in_lookaside', 'prepared updates in cache overflow'),
+    TxnStat('txn_prepared_updates_resolved', 'prepared resolved updates'),
```

Comment:
suggest renaming description to "prepared updates resolved"

Reply:
Do you think we need a separate commited/rollbacked rather than single resolved ?

Reply:
We need whether prepared updates are resolved or not, to check whether we missed to resolve any prepared update, not interested in mechanism (commit/rollback) resolution.